client/state/products-list/selectors.js

```diff
@@ -1,3 +1,7 @@
+import { property } from 'lodash';
+
+export const getProductsList = property( 'productsList.items' );
```

Comment:
> Why would we do this, when we can use test data?

well that was my point. what test data do I use? I end up creating data to match the implementation of the selector in order to test that the implementation of the selector matches the test data, no?

I'm pretty opposed to testing that the code and structure we write matches the code and structure we write. I'm much more excited about testing behaviors.

```js
// file.js
export const CONSTANT = 5;

// test.js
import { CONSTANT } from 'file'

assert( CONSTANT === 5 );
```